src/main/java/gov/usds/case_issues/model/CaseSnoozeFilter.java

```diff
@@ -0,0 +1,15 @@
+package gov.usds.case_issues.model;
+
+/**
+ * Categories of filter that a user of the general case list API might request.
+ */
+public enum CaseSnoozeFilter {
```

Comment:
Nothing in this file has a name that I like. Suggestions welcome, though I probably won't like them either. Naming sucks.

Reply:
The `ACTIVE` filter seems like a nice hack to maintain the current state of our app. But long term it seems like a more flexible API if we support multiple `CaseSnoozeFilters` instead of having a combined filter

Reply:
Long term it definitely is. Right now, I am torn between forward compatibility and practicality—of the seven possible combinations of those three states, we still only support three at the query level.

Reply:
Yeah I think my vote is this works for now but we should fix it and work a naming as we work on the more complex filtering

Reply:
Agree—also, once we have a more flexible query system we could actually include `CLOSED` as a case type, which would make me happy.